### PR TITLE
Add iziswap on zksync to `dex.trades`

### PR DESCRIPTION
**Pull request by thetroyharris**

### CODE REVIEW

**Comment by thetroyharris** [comment]
Ready for review @Hosuke @jeff-dude

File: models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            , CASE WHEN sellXEarnY THEN tokenX ELSE tokenY END AS token_sold_address
+            , CASE WHEN sellXEarnY THEN amountY ELSE amountX END AS token_bought_amount_raw 
+            , CASE WHEN sellXEarnY THEN amountX ELSE amountY END AS token_sold_amount_raw
+        FROM {{ source('iziswap_v1_zksync', 'iZiSwapPool_evt_Swap') }}
+        {% if is_incremental() %}
+        WHERE {{incremental_predicate('evt_block_time')}}
+        {% else %}
+        WHERE evt_block_time >= TIMESTAMP '{{iziswap_start_date}}'
+        {% endif %}
+        UNION
```
Comment (Hosuke) [inline models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql:34]: You can separate version `1` and `2` in two files if you want.

File: models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    , CAST(DATE_TRUNC('month', block_time) AS DATE) AS block_month
+    , CAST(DATE_TRUNC('day', block_time) AS DATE) AS block_date
+    , block_time
+    , block_number
+    , token_bought_amount_raw
+    , token_sold_amount_raw
+    , token_bought_address
+    , token_sold_address
+    , CAST(NULL AS VARBINARY) AS taker
+    , CAST(NULL AS VARBINARY) AS maker
```
Comment (Hosuke) [inline models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql:67]: I think we may need to try to fill these two fields.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql (hunk 2 above)
Comment (thetroyharris) [reply models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql]: Done

File: models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql (hunk 1 above)
Comment (thetroyharris) [reply models/_sector/dex/trades/zksync/platforms/iziswap_zksync_base_trades.sql]: Done

**Comment by jeff-dude** [comment]
thanks for the changes, lgtm

**Review by Hosuke — APPROVED** [review]

**Pull request merged**